packages/astro/src/vite-plugin-markdown/index.ts

```diff
@@ -203,10 +207,18 @@ ${tsResult}`;
 					sourcemap: false,
 					sourcefile: id,
 				});
+
+				const astroMetadata: AstroPluginMetadata['astro'] = {
+					clientOnlyComponents: transformResult.clientOnlyComponents,
+					hydratedComponents: transformResult.hydratedComponents,
+					scripts: transformResult.scripts
```

Comment:
Curious why we don't also need styles here? Is there an opportunity for this change to simplify our custom style handling?

Reply:
This is what gets fed into the Rollup `input` option. Those are only JS. Styles are are built by `vite:css-post` as part of building the JS. Rollup doesn't know about CSS, it only speaks JS.